docs/tutorials/model-development/5.-SymbolicOperations_and_CLBM.md

```diff
@@ -328,54 +328,8 @@ git clone https://github.com/CFD-GO/TCLB_tools.git
 ```
 
 Then open the `TCLB_tools/Python/symbolic_tools/` directory in PyCharm and create a new pycharm-project.
-Next, create a folder `tutorials` and add a new python script `tutorial_cm_eq_derivation.py`
-Run the script.
-
-```py
-from SymbolicCollisions.core.printers import print_as_vector
-from sympy.matrices import Matrix
-from sympy import Symbol
-from SymbolicCollisions.core.ContinousCMTransforms import ContinousCMTransforms, get_mom_vector_from_continuous_def
-from SymbolicCollisions.core.cm_symbols import \
-    F3D, dzeta3D, u3D, rho
-
-from SymbolicCollisions.core.cm_symbols import rho, moments_dict
-import time
-
-
-lattice = 'D2Q9'
-ccmt = ContinousCMTransforms(dzeta3D, u3D, F3D, rho)
-
-start = time.process_time()
-
-print('\n\n// === continous cm === \n ')
-
-# to calculate particular moment
-row = moments_dict['D2Q9'][0]
-moment = ccmt.get_cm(row, ccmt.get_Maxwellian_DF)
-print_as_vector(Matrix([moment]), 'particular_moment')
-
-
-print('\n//Force -> Force_cm - from continous definition: \n'
-      'k_mn = integrate(fun, (x, -oo, oo), (y, -oo, oo)) \n'
-      'where fun = forceM(rho,u,x,y) *(x-ux)^m *(y-uy)^n *(z-uz)^o ')
-F_cm = get_mom_vector_from_continuous_def(ccmt.get_force_He_MB,
-                                          continuous_transformation=ccmt.get_cm,
-                                          moments_order=moments_dict[lattice])
-print_as_vector(F_cm, 'F_cm')
-
-
-print('\n//population_eq -> cm_eq - from continous definition: \n'
-      'k_mn = integrate(fun, (x, -oo, oo), (y, -oo, oo)) \n'
-      'where fun = fM(rho,u,x,y) *(x-ux)^m *(y-uy)^n *(z-uz)^o ')
-cm_eq = get_mom_vector_from_continuous_def(ccmt.get_Maxwellian_DF,
-                                           continuous_transformation=ccmt.get_cm,
-                                           moments_order=moments_dict[lattice])
-print_as_vector(cm_eq, 'cm_eq')
-
-
-print(f'\n\n Done in {time.process_time() - start} [s].')
-```
+Next, go `tutorials` folder and open the `tutorial_cm_eq_derivation.py` script.
```

Comment:
link to `https://github.com/CFD-GO/TCLB_tools/blob/master/Python/symbolic_tools/SymbolicCollisions/tutorials/tutorial_cm_eq_derivation.py`